png.BUILD

```diff
@@ -33,7 +42,13 @@ genrule(
 cc_library(
     name = "png",
     srcs = [prefix_dir + "/" + source for source in PNG_SOURCES],
-    hdrs = glob(["**/*.h"]) + [":configure"],
+    # Don't run configure on Windows
+    hdrs = glob(["**/*.h"]) + select({
+        ":windows": [],
+        "//conditions:default": [":configure"],
+    }),
+    # png needs zlib to build
+    deps = ["@zlib_archive//:zlib"],
     includes = [prefix_dir],
     linkopts = ["-lz"],
```

Comment:
Since we already depend on `@zlib_archive//:zlib`, how about just remove this flag?